news/2017-01-19-bean-validation-2-0-progress-report.adoc

```diff
@@ -0,0 +1,235 @@
+allowing providers of container types to bundle corresponding extractors with their library and making them automatically available to you.
+
+Validation of container elements is by far the most complex feature and we'd like to gather some more feedback on it before committing to it.
+Hence its current proposal is added as http://beanvalidation.org/latest-draft/spec/#appendix-value-extraction[an appendix] to the spec draft.
+We are eager to learn about your thoughts and feedback in general, but it's especially important for this issue due to its complexity.
+We've compiled a list of http://beanvalidation.org/latest-draft/spec/#_open_questions[open questions] around this proposal.
+If you have thoughts on any of those, please make sure to let us know, e.g. by commenting below.
+
+== Other improvements
+
+While support for JSR 310 and validation of container elements have been the largest features we've been working on so far,
+there are some more smaller, yet very useful improvements.
+
+E.g. all the built-in constraints are http://docs.oracle.com/javase/tutorial/java/annotations/repeating.htmlhttps://hibernate.atlassian.net/browse/BVAL-497[repeatable annotations] now, allowing to define them several times without requiring the explicit `@List` annotation ([BVAL-497]):
+
+[source,java]
+----
+@ZipCode(countryCode = "fr", groups = Default.class, message = "zip code is not valid")
+@ZipCode(
+    countryCode = "fr",
+    groups = SuperUser.class,
+    message = "zip code invalid. Requires overriding before saving."
+)
+private String zipCode;
+----
+
+`ConstraintValidator#initialize()` has an empty default implementation now (https://hibernate.atlassian.net/browse/BVAL-555[BVAL-555]),
+simplifying the implementation of constraint validators that don't need to access any constraint attributes.
+You can simply omit the `initialize()` method:
+
+[source,java]
+----
+public class AssertTrueValidator implements ConstraintValidator<AssertTrue, Boolean> {
+
+    @Override
+    public boolean isValid(Boolean bool, ConstraintValidatorContext constraintValidatorContext) {
+        return bool == null || bool;
+    }
+}
+----
+
+Another nice improvement is the usage of actual parameter names when reporting constraint violations for constraints on method or constructor parameters (https://hibernate.atlassian.net/browse/BVAL-498[BVAL-498]).
+Provided you have enabled reflective parameter name access during compilation (using `-parameters` javac option),
+`Path.Node#getName()` will return the actual parameter name instead of "arg0", "arg1" for parameter nodes.
+
+== Next steps
+
+We are currently working on picking up some loose ends around the proposal for container element validation.
+Once that is done, we feel it is the right time to put out an Alpha1 release of Bean Validation 2.0 and post it for Early Draft Review to the JCP.
```

Comment:
Hum really, we are waiting for feedback and another new cycle before pushing it? I'm against that. Pushing it in the JSP is about feedback.